deploy/helm/opa-operator/crds/crds.yaml

```diff
@@ -459,6 +507,7 @@ spec:
                                           - ERROR
                                           - FATAL
                                           - NONE
+                                          - null
```

Comment:
Just FYI that this is introduced by kube 2. I *think* it doesn't hurt to have it here